src/main/java/io/neonbee/job/JobVerticle.java

```diff
@@ -74,12 +74,20 @@ public String getName() {
     @Override
     public void start() {
         if (!NeonBee.get(getVertx()).getOptions().shouldDisableJobScheduling()) {
+            if (schedule.isPeriodic() && isTemporalAdjusterInvalid()) {
+                throw new IllegalStateException("JobVerticle was configured with an invalid JobSchedule");
```

Comment:
Very technical error message with class names, is ir better to just call it an "invalid schedule" and explain why it is invalid

Reply:
done